<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Observability Program</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>11</number>
    </property>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="log_add">
        <property name="text">
         <string>Add New Log</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="log_edit">
        <property name="text">
         <string>Edit Log</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="log_display">
        <property name="text">
         <string>Display Log</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="log_del">
        <property name="text">
         <string>Delete Log</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QTableWidget" name="log_table">
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column/>
      <column/>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Notes Of The Star</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTextEdit" name="log_note">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCheckBox" name="log_editbox">
          <property name="text">
           <string>Editable</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="log_save">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="log_cancel">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="log_label">
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionStar_List">
   <property name="text">
    <string>Star List</string>
   </property>
  </action>
  <action name="action_load_stars">
   <property name="text">
    <string>Load Stars</string>
   </property>
  </action>
  <action name="action_load_observatories">
   <property name="text">
    <string>Load Observatories</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
